www/templates/404.html

```diff
@@ -0,0 +1,27 @@
+<ion-view>
+	<ion-content class="padding">
+		<!-- <p>Page not found.</p> -->
+		
+		<div class="msg-404">Page not found</div>
+		
+		<p>The page cannot be found. Please leave your first name, last name, email, and comments. We will get back to you soon.</p>
+		<div class="list">
+
+		  <label class="item item-input">
+		    <input type="text" placeholder="First Name">
+		  </label>
+		  <label class="item item-input">
+		    <input type="text" placeholder="Last Name">
```

Comment:
I am open to whatever you both decide. I can also just put a name field for people to put their whole first and last name in too.